uqcsbot/scripts/yelling.py

```diff
@@ -2,6 +2,7 @@
 
 from random import choice, random
 from re import sub, UNICODE
+import re
```

Comment:
`import re` should go first in imports because it's a global import (the whole package).